pkgs/development/libraries/physics/geant4/datasets.nix

```diff
@@ -27,6 +37,7 @@ let
         homepage = "https://geant4.web.cern.ch/support/download";
         license = licenses.g4sl;
         platforms = platforms.all;
+        inherit priority;
```

Comment:
This I haven't seen before. What effect does different values have?

Reply:
It makes it so that the files that are named the same between the packages do not conflict.  Global default value is 5.   0 is highest priority.

Reply:
Alright, lgtm then.

Reply:
Setting `meta.priority` will not fix the underlying issue. Let's not do that.